xcm/xcm-simulator/example/src/parachain.rs

```diff
@@ -49,7 +49,7 @@ use xcm_executor::{
 };
 
 pub type SovereignAccountOf = (
-	SiblingParachainConvertsVia<ParaId, AccountId>,
+	SiblingParachainConvertsVia<Sibling, AccountId>,
```

Comment:
Is this intended?

Reply:
yes, `ParaId` has different encoding then `Sibling`,
and e.g. everywhere in Cumulus there is used `SiblingParachainConvertsVia<Sibling, ` and also everywhere `ParaId` is used by relaychain cfg like `ChildParachainConvertsVia<ParaId, `

I found it just by accident because of: https://github.com/paritytech/cumulus/pull/2167/files#diff-07287c62dd91150d975f7504bd14cc62d9889c44f7ba8ca44c30c03b05d5b071L389-R421

Reply:
> yes, ParaId has different encoding then Sibling,

It doesnt. But the change is ok - just slightly more explicit about the intent.